esmvaltool/utils/recipe_test_workflow/doc/source/user_guide/workflow.rst

```diff
@@ -27,8 +27,9 @@ The |RTW| performs the following steps:
   :Executes:
      The ``configure.py`` script from the |Rose| app
   :Details:
-     Runs once at the start of the workflow, immediately after the successful
-     completion of the ``install_cold`` job
+     Runs once at the beginning of the workflow, after the install_cold task,
+     and then at the start of each cycle, after the successful completion
+     of the previous cycle's compare job
```

Comment:
The configure task must only run once (running the configure task again will not change anything, so there's no point in using resources to run it again).

Reply:
Discussed - new task to be made to address this.

Reply:
Would it be possible to revert this change and deal with updating the documentation in the new issue?

Reply:
This is a description of what is currently going on in the workflow. I think we should keep this as is and then update it when we change the behaviour. Otherwise, the documentation will not be reflecting the behaviour.